index.js

```diff
@@ -0,0 +1,37 @@
+var program = require('commander');
+var requireDir = require('require-dir');
+var package = require(__dirname + '/package.json');
+
+program.version(package.version)
+  .option('-D, --debug', 'enable debugging and debug output')
+  .option('-q, --quiet', 'quiet output, except for errors')
+  .option('-v, --verbose', 'verbose output')
+  .option('-j, --json', 'JSON data output')
+  .option('--nocolors', 'no ANSI colors in JSON output');
+
+// Common command init wrapper with error reporting.
+function init (cmd) {
+  return function () {
+    var args = Array.prototype.slice.call(arguments, 0);
```

Comment:
This is declared then only used once right after... will it overflow line-length to just put it right into the `.apply()` call?